### ISSUE TITLE
[Feature] Add tooltip to location buttons

### ISSUE THREAD

**Issue by sphh** [linuxmint/nemo#2911]
```
 * Nemo version: 5.2.4
 * Is issue with desktop or windowed nemo?: Windowed
 * Distribution: Mint 20.2
 ```

**Issue**
If you have the current location displayed as buttons, depending on the available space only the last directory names are displayed. Let us assume that you have the following paths: `~/a/b/c/very long calculations/some random results` and `~/x/y/z/very long calculations/some random results`. If the window is not large enough, only the last directory names are displayed: `[very long calculations][some random results]`. It is not possible to decide, if you are working in `~/a/b/c/` or `~/x/y/z/`.

To determine, which directory is actually displayed, you have to scroll the buttons with the arrows. But then the last directory names will disappear.

It would be helpful, that the button shows a tooltip with the whole path of the directory it represents (maybe even multiline, if there is not enough space). Thus it is quick and easy to check the actual path: Just move the mouse cursor over the button and you immediately know, where you are!

**Steps to reproduce**
1. Create two paths with long directory names, e.g. `~/a/b/c/very long calculations/some random results` and `~/x/y/z/very long calculations/some random results`.
2. Open two nemo windows and in each one navigate to `some random results`. Make the window small enough (if it not already too small to show more than the last two directory names), so that the location buttons for `~/a/c/d` and `~/x/y/z` are not shown.
3. Switching between those two windows, you don't know anymore, where you are.
4. Moving the mouse cursor over the location buttons will not display the full path.

**Expected behaviour**
Positioning the cursor over the location buttons should display the full path to the directory represented by the button.

**Other information**
As a workaround it is possible to activate `Show the full path in the title bar and tab bars`. But with long paths, these often get truncated showing only parts of the path.

Another workaround is to work with tabs instead of separate windows, because the tab already shows a tooltip of the directory.

**Comment by Jeremy7701** [comment]
There is an optional toolbar button to show quite a long path:-

![Screenshot from 2022-01-16 16-09-14](https://user-images.githubusercontent.com/52585119/149667951-09bbb1e7-0356-4d34-b8a0-7a99dfdbb8cb.png)

**Comment by sphh** [comment]
Yes, you are absolutely correct. I should have mentioned this under the section ‘Other information’. But if there is not enough space, the path displayed in the edit field also overflows on the left side hiding a relevant part of it:
![Screenshot from 2022-01-16 17-17-23](https://user-images.githubusercontent.com/4856462/149668255-4228274d-d97e-43a1-a36a-833e8e953ce6.png)

And BTW it requires to click onto the toggle button to show it and then it is another click to hide it again.

**Comment by sphh** [comment]
I just tested it with Nemo 6.6.3 and everything said above is still valid.

It would be great, if the directory buttons would grow a tooltip with the whole path.

**Comment by JosephMcc** [comment]
Considering this has been implemented in 4 years, I wouldn't hold my breath. Feature requests now belong here: https://github.com/orgs/linuxmint/discussions

**Comment by sphh** [comment]
@JosephMcc, I am ok, that this is transferred to [Linux Mint Discussion](https://github.com/org/linuxmint/discussions), since the policy has [changed](https://github.com/linuxmint/nemo/issues/new/choose#_R_8vr9b_).

But if you close it, it would have been an act of courtesy to also transfer the contents over to the discussion. I cannot find it there, so I ask you to transfer the issue(s) you close this way over to discussions. Thanks.
